src/sentry/integrations/gitlab/integration.py

```diff
@@ -28,11 +30,39 @@
 )
 
 
+API_ERRORS = {
+    404: 'Gitlab returned a 404 Not Found error. If this repository exists, ensure'
```

Comment:
We might have to change this - this makes more sense for the GH Apps flow of giving permissions to certain repos. I think this might want to be similar to the old messaging since it's still using OAuth? We can also double back on this